### PR TITLE
Create a build GitHub workflow

### PR DESCRIPTION
**Pull request by swaroop9**
Added a build.yml file to build and test the changes on push and pull request events.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Build
+
+on:
+  pull_request:
+    paths-ignore:
+     - 'README.md'
+    branches: 
+      - master
```
Comment (ASaiAnudeep) [inline .github/workflows/build.yml:8]: Default branch is `main`

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  push:
+    paths-ignore:
+     - 'README.md'
+    branches: 
+      - master
```
Comment (ASaiAnudeep) [inline .github/workflows/build.yml:13]: Same here.
Default branch is `main`

Comment (swaroop9) [reply .github/workflows/build.yml]: Done

File: .github/workflows/build.yml (hunk 1 above)
Comment (swaroop9) [reply .github/workflows/build.yml]: Done

**Pull request merged**